integration/proxy/proxy_test.go

```diff
@@ -1162,19 +1161,25 @@ func TestALPNProxyHTTPProxyBasicAuthDial(t *testing.T) {
 	t.Setenv("http_proxy", helpers.MakeProxyAddr(user, pass, proxyURL.Host))
 
 	rcProxyAddr := net.JoinHostPort(rcAddr, helpers.PortStr(t, rc.Web))
-	require.Zero(t, ph.Count())
 	nodeCfg := makeNodeConfig("node1", rcProxyAddr)
 	nodeCfg.Log = log
-	_, err = rc.StartNode(nodeCfg)
-	require.Error(t, err)
 
+	require.Zero(t, ph.Count())
```

Comment:
Seems not worth checking for until the node has been started?

Reply:
Sanity check, but yes it should always be 0 I can remove